commons-dlp/src/main/java/org/exoplatform/commons/dlp/service/impl/DlpPermissionsServiceImpl.java

```diff
@@ -100,7 +98,14 @@ public void removeDlpPermissionsPagesAndNavigations(String oldPermissions) throw
     for (String permission : permissionsList){
       NavigationContext nav = navigationService.loadNavigation(SiteKey.group(permission));
       if (nav != null) {
-        navigationService.destroyNavigation(nav);
+        NodeContext node = navigationService.loadNode(NodeModel.SELF_MODEL, nav, Scope.ALL, null);
+        if(node.get(DLP_QUARANTINE) != null){
+          if(node.getNodeSize() == 1) {
+            navigationService.destroyNavigation(nav);
+          } else {
+            node.removeNode(DLP_QUARANTINE);
+          }
+        }
         String pageKey = PortalConfig.GROUP_TYPE + "::" + permission +"::dlp-quarantine";
```

Comment:
I think that this block should be into if(node.get(DLP_QUARANTINE) != null) WDYT ?
```suggestion
        String dlpPageKey = PortalConfig.GROUP_TYPE + "::" + permission +"::" + DLP_QUARANTINE;
```